Upstream-Status: Pending

This patch avoids following issues with automake 1.12:

| automake: warnings are treated as errors
| /srv/home/nitin/builds/build-gcc47/tmp/sysroots/x86_64-linux/usr/share/automake-1.12/am/ltlibrary.am: warning: 'libtasn1.la': linking libtool libraries using a non-POSIX
| /srv/home/nitin/builds/build-gcc47/tmp/sysroots/x86_64-linux/usr/share/automake-1.12/am/ltlibrary.am: archiver requires 'AM_PROG_AR' in 'configure.ac'

Signed-Off-By: Nitin A Kamble <nitin.a.kamble@intel.com>
2012/05/02


Index: libtasn1-2.11/configure.ac
===================================================================
--- libtasn1-2.11.orig/configure.ac
+++ libtasn1-2.11/configure.ac
@@ -38,6 +38,11 @@ lgl_EARLY
 AC_PROG_YACC
 AC_PROG_LN_S
 
+# automake 1.12 seems to require this, but automake 1.11 doesn't recognize it
+m4_pattern_allow([AM_PROG_AR])
+AM_PROG_AR
+
+
 dnl Checks for programs.
 AC_PROG_INSTALL
 AM_MISSING_PROG(HELP2MAN, help2man, $missing_dir)
